assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -53,276 +65,517 @@ export const PRE_PUBLISH_MESSAGE_TYPES = {
+            )}
+          </li>
+        </DashedList>
       ),
     },
     FONT_TOO_SMALL: {
-      MAIN_TEXT: __('Font size too small', 'web-stories'),
-      HELPER_TEXT: sprintf(
-        /* translators: 1: minimum font size. */
-        __('Text should have size %d or larger.', 'web-stories'),
-        MIN_FONT_SIZE
+      MAIN_TEXT: __('Increase font size', 'web-stories'),
+      HELPER_TEXT: (
+        <DashedList>
+          <li>
+            {sprintf(
+              /* translators: %d: minimum font size. */
+              __(
+                'Ensure legibility by selecting text size %d or greater',
+                'web-stories'
+              ),
+              MIN_FONT_SIZE
+            )}
+          </li>
+        </DashedList>
       ),
     },
     LOW_IMAGE_RESOLUTION: {
-      MAIN_TEXT: __('Low image resolution', 'web-stories'),
-      HELPER_TEXT: sprintf(
-        /* translators: 1: minimum image size width x minimum image size height. */
-        __(
-          'Use %s for full screen images. Use similar pixel density for cropped images.',
-          'web-stories'
-        ),
-        `${IMAGE_SIZE_WIDTH}x${IMAGE_SIZE_HEIGHT}px`
+      MAIN_TEXT: __('Select image with higher resolution', 'web-stories'),
+      HELPER_TEXT: (
+        <DashedList>
+          <li>
+            {sprintf(
+              /* translators: %s: minimum image size width x minimum image size height. */
+              __('Use %s for a full-screen image', 'web-stories'),
+              `${IMAGE_SIZE_WIDTH}x${IMAGE_SIZE_HEIGHT}px`
+            )}
+          </li>
+          <li>
+            {__(
+              'Consider similar pixel density for cropped images',
+              'web-stories'
+            )}
+          </li>
+        </DashedList>
       ),
     },
     MISSING_CAPTIONS: {
-      MAIN_TEXT: __('Video missing captions', 'web-stories'),
-      HELPER_TEXT: __(
-        'Add captions to your video. Captions help keep your audience engaged, even when they can’t listen to the audio.',
-        'web-stories'
+      MAIN_TEXT: __('Add captions to video', 'web-stories'),
+      HELPER_TEXT: (
+        <DashedList>
+          <li>
+            {__(
+              "Keep the audience engaged even when they can't listen to the audio",
+              'web-stories'
+            )}
+          </li>
+        </DashedList>
       ),
     },
     MISSING_VIDEO_DESCRIPTION: {
-      MAIN_TEXT: __('Video missing description', 'web-stories'),
-      HELPER_TEXT: __(
-        'Add a video description. Descriptions help with indexability and accessibility (for videos without captions). Include any burned-in text inside of the video.',
-        'web-stories'
+      MAIN_TEXT: __('Add video description', 'web-stories'),
+      HELPER_TEXT: (
+        <DashedList>
+          <li>
+            {__(
+              'Improves indexability and accessibility (for videos without captions)<br/>- Include any text or captions that are already a rendered part of the video',
```

Comment:
instead of a `<br />` tag and a dash, could you move the second phrase starting with `Include` to be a second list item?

Reply:
good catch!